charts/influxdb/files/backup-retention-script.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash
+
+# Enable unofficial bash strict mode
+set -o errexit
+# set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+set -o xtrace
```

Comment:
Maybe this is sufficient
```suggestion
set -eu -o pipefail
```

Reply:
or just `set -e`